webui/react/src/pages/TrialsComparison/TrialsComparison.tsx

```diff
@@ -31,7 +31,9 @@ interface Props {
 }
 
 const TrialsComparison: React.FC<Props> = ({ projectId }) => {
-  const tableSettingsHook = useSettings<InteractiveTableSettings>(trialsTableSettingsConfig);
+  const tableSettingsHook = useSettings<InteractiveTableSettings>(
+    trialsTableSettingsConfig(projectId),
```

Comment:
I think we might want to memoize this, since `config` a dependency of various things in `useSettings`